config/templates/cluster.yaml

```diff
@@ -469,6 +469,12 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 #     # See https://github.com/coreos/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
+#    # When enabled it will grant sts:assumeRole permission to everyrole in the IAM Worker Role.
+#    # if you specify workerNamedIAMRole the role created will not suffix the random id at the end
+#    # which is useful to update trust policy in target IAM roles that container wants to use through kube2iam.
+#    kube2IamSupport:
+#      enabled: true
+#      workerNamedIAMRole: "yournamedrole"
```

Comment:
@c-knowles one thing i can do is to change the namedRole to something like {{stack-name}}-IAMRoleWorker-{{suffixRoleName}, and then change  workerNamedIAMRole to workerNamedIAMRoleSuffix and set a sensible default like kube2iam, i would prefer to give the users the option to set the naming it makes most sense in their scenario.